data/hai/hai missions.txt

```diff
@@ -833,11 +836,15 @@ mission "Nanachi 1"
 	to offer
 		random < 15
 		or
-			has "event: hai-human resolution announced"
 			and
-				"net worth" > 2000000000
+				"net worth" > 200000000
+				"combat rating" > 1200
+				has "event: hai-human resolution announced"
+			and
+				"net worth" > 200000000
```

Comment:
I think we should keep the old 20M but since there is already a PR for that you could suggest this change of it not popping in the middle of it there?

Reply:
~~Just merge Z's first and I'll resolve conflicts.~~

Reply:
I just incorporated the changes to this section into #7887 . It's organized slightly differently (removes the duplication between the `or` statements, but accomplishes the same thing in fewer lines. )

Reply:
Great then!
Still feel like we can go back to 20M but thats a discussion for that other PR

Reply:
> ~~Just merge Z's first and I'll resolve conflicts.~~

As it stands, it looks like your version just does the same thing as Zitchas' is except 10x the net worth (for no reason?) and the condition set is unnecessarily complex.

Reply:
> > ~~Just merge Z's first and I'll resolve conflicts.~~
> 
> As it stands, it looks like your version just does the same thing as Zitchas' is except 10x the net worth (for no reason?) and the condition set is unnecessarily complex.

This PR should perhaps just not touch this and you should express what you want in Zitchas' PR; there's no point merging that one if this is just going to change the whole condition set anyway, but that one was open first, so unless Zitchas' proposal is unacceptable to you, it seems courteous to just make this change over there.

Reply:
Yeah just remove this change from this PR and let the other PR worry about it